ficed gcc 11: avb_slot_verify.c:  error: ‘found’ may be used uninitialized in this function
---

--- a/lib/avb/libavb/avb_slot_verify.c	2021-10-01 23:31:29.953891262 +0000
+++ a/lib/avb/libavb/avb_slot_verify.c	2021-10-01 23:31:22.801912189 +0000
@@ -296,7 +296,7 @@
   bool image_preloaded = false;
   uint8_t* digest;
   size_t digest_len;
-  const char* found;
+  const char* found = NULL;
   uint64_t image_size;
   size_t expected_digest_len = 0;
   uint8_t expected_digest_buf[AVB_SHA512_DIGEST_SIZE];
